main-backend/api.py

```
@@ -0,0 +1,35 @@
+
+# Request model
+class Message(BaseModel):
+    message: str
+
+
+#  REST API Endpoint
+@app.post("/chat")
+async def chat(msg: Message):
+    try:
+        response = get_response(msg.message)
+        return {"response": response}
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)